components/BottomSheet/BUILD

```diff
@@ -45,6 +47,31 @@ mdc_objc_library(
     visibility = ["//visibility:private"],
 )
 
+mdc_objc_library(
+    name = "app_tests_lib",
+    testonly = 1,
+    srcs = glob(["tests/app/*.m"]),
+    hdrs = glob(["tests/app/*.h"]),
+    sdk_frameworks = [
+        "UIKit",
+        "XCTest",
+    ],
+    deps = [":BottomSheet"],
+    visibility = ["//visibility:private"],
+)
+
+# These are separate from the regular unit test suite because we want to specify
+# test_host so we can exercise tests which depend on UIKit functionality like
+# UIViewController presentation logic.
+ios_unit_test_suite(
+    name = "app_tests",
+    deps = [":app_tests_lib"],
+    minimum_os_version = "8.0",
+    size = "medium",
+    test_host = "@build_bazel_rules_apple//apple/testing/default_host/ios",
+    runners = DEFAULT_IOS_RUNNER_TARGETS,
+)
```

Comment:
Can we promote this to the .bzl file so that other targets can reuse it?

Reply:
Deleted prior comment down below because I think if you move this to the .bzl file we won't need to introduce the //components/testing/runners target.

Reply:
Split out into https://github.com/material-components/material-components-ios/pull/3422.